pkg/apis/atlassian/v1/nodegroup_types.go

```diff
@@ -39,6 +39,10 @@ type NodeGroupSpec struct {
 
 	// SkipPreTerminationChecks is an optional flag to skip pre-termination checks during cycling
 	SkipPreTerminationChecks bool `json:"skipPreTerminationChecks,omitempty"`
+
+    // Priority controls the ordering of CNR creation for this NodeGroup.
+    // Lower values are higher priority. 0 runs first, then 1, 2, ...
```

Comment:
nit: fix comment to include example negative number, or reference negative numbers are possible